EdgeMultiplay/Scripts/EdgeManager.cs

```diff
@@ -201,7 +210,16 @@ public async Task ConnectToServer(bool useAnyCarrierNetwork = true, bool useFall
                     integration.UseWifiOnly(useAnyCarrierNetwork);
                     integration.useFallbackLocation = useFallBackLocation;
                     wsClient = new MobiledgeXWebSocketClient();
-                    await integration.RegisterAndFindCloudlet();
+                    try
+                    {
+                        await integration.RegisterAndFindCloudlet();
+                    }
+                    catch (LocationException)
+                    {
+                        MobiledgeXIntegration.LocationFromIPAddress location = await MobiledgeXIntegration.GetLocationFromIP();
```

Comment:
I think we should not catch the LocationException here. I think we should just kick it down to the application developer. If their application doesn't necessarily need the fastest edge server, they may not want to use GetLocationFromIP. Or they may just want different behavior in their application if the user rejects location services.